service/test/ConstConfigIOGroupTest.cpp

```diff
@@ -0,0 +1,658 @@
+/*
+ * Copyright (c) 2015 - 2022, Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include "ConstConfigIOGroup.hpp"
+
+#include <cerrno>
+#include <set>
+#include <string>
+#include <fstream>
+#include <iostream>
+#include <regex>
+#include <unistd.h>
+
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+#include "geopm_topo.h"  // for geopm_domain_e
+#include "geopm_test.hpp" // for GEOPM_EXPECT_THROW_MESSAGE
+
+#include "geopm/Helper.hpp"
+#include "geopm/Agg.hpp"
+#include "geopm/Exception.hpp"
+#include "geopm/PlatformIO.hpp"
+#include "geopm/PlatformTopo.hpp"
+
+using geopm::ConstConfigIOGroup;
+using geopm::Exception;
+using geopm::PlatformIO;
+using geopm::PlatformTopo;
+
+class ConstConfigIOGroupTest : public::testing::Test
+{
+    protected:
+        static const std::string M_CONFIG_FILE_PATH;
+        static void create_config_file(const std::string &config);
+};
+
+const std::string ConstConfigIOGroupTest::M_CONFIG_FILE_PATH =
+        "/tmp/const_config_test.json";
```

Comment:
```suggestion
        "const_config_test.json";
```
Is there a need to write these to tmp as opposed to the CWD?  If we put them in tmp, we may have other GEOPM users who are also writing to the same spot.  If 2 people are running make check at the same time, this is a problem.

Reply:
Thanks, will fix.